src/shared/interfaces/ProfileFieldInterfaces.tsx

```diff
@@ -0,0 +1,17 @@
+export type fields = {
+  title: string;
+  content: contentField;
+};
+export interface UserPageFieldProps {
+  data: fields;
+  isLoading: boolean;
+  isError: boolean;
+}
+export interface UserPageFieldInterface {
```

Comment:
export interface UserPageFieldProps<T> {
  data: T;
  isLoading: boolean;
  isError: boolean;
}

Reply:
you could add generic type here